hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/response/OMClientResponse.java

```diff
@@ -85,8 +85,15 @@ public void checkStatusNotOK() {
    */
   public void checkAndUpdateDB(OMMetadataManager omMetadataManager,
       BatchOperation batchOperation) throws IOException {
-    if (omResponse.getStatus() == OzoneManagerProtocolProtos.Status.OK) {
+    switch (omResponse.getStatus()) {
+    case PARTIAL_DELETE:
+    case PARTIAL_RENAME:
+    case OK:
       addToDBBatch(omMetadataManager, batchOperation);
+      break;
+    default:
+      // do nothing
+      break;
```

Comment:
> Thanks for catching this. I think:
> 
> 1. It should be done in a separate Jira/PR, as it's a different bug.
> 2. It would be nice to add a test case for the bug this is fixing.
> 3. Keep it consistent with `OMKeysDeleteResponse`, which already handles `PARTIAL_DELETE`:
>    https://github.com/apache/ozone/blob/68c5ac5df4fbc0edd7114394112fa696a3dc9229/hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/response/key/OMKeysDeleteResponse.java#L63-L70
>    
>    By keeping it consistent I mean: either remove that overridden method and handle it in `OMClientResponse`, or override in `OMKeysRenameResponse`, too.  I slightly prefer this latter solution.

got it.

Reply:
I may be missing something, but I don't see such failing test case on `master` (i.e. without the fix).